pyrevolve/revolve_bot/revolve_bot.py

```diff
@@ -255,15 +253,17 @@ def render_brain(self, img_path):
         """
         if self._brain is None:
             raise RuntimeError('Brain not initialized')
-        else:
+        elif isinstance(self._brain, BrainNN):
             try:
                 brain_graph = BrainGraph(self._brain, img_path)
-                brain_graph.brain_to_graph()
+                brain_graph.brain_to_graph(True)
                 brain_graph.save_graph()
             except Exception as e:
                 print('Failed rendering brain. Exception:')
                 print(e)
                 print(traceback.format_exc())
+        else:
+            print('Brain not initialized')
```

Comment:
raise runtime exception
   `"Brain {type(self._brain)} image rendering not supported"`